samples/automl/automlTranslationDataset.js

```diff
@@ -106,24 +108,26 @@ function listDatasets(projectId, computeRegion, filter_) {
       const dataset = responses[0];
 
       // Display the dataset information.
-      console.log('List of datasets:');
-      for (let i = 0; i < dataset.length; i += 1) {
-        console.log('Dataset name:', dataset[i].name);
-        console.log('Dataset id: ', dataset[i].name.split('/').pop(-1));
-        console.log('Dataset display name: ', dataset[i].displayName);
-        console.log('Dataset example count: ', dataset[i].exampleCount);
-        console.log('Translation dataset specification:');
+      console.log(`List of datasets:`);
+      for (let i of dataset) {
+        console.log(`Dataset name: ${i.name}`);

```

Comment:
Replace the `Dataset` in the **string** (your second arguments to `console.log` were mostly fine as-is).

Apply this comment throughout this PR.